docs/jobtemplates.rst

```diff
@@ -0,0 +1,147 @@
+Defining host templates
+=======================
+
+It is possible to define your own host templates in addition to the ones shipped
+with the package. A template file should have the same filename as the UNIX
+command ``hostname`` returns to be detected automatically. Otherwise you can
+point MDBenchmark to a specific template by providing its name via the
+``--host`` option.
+
+Assuming you created a new host template in your home directory ``~/.config/MDBenchmark/my_custom_hostfile``::
+
+    mdbenchmark generate --host my_custom_hostfile
+
+Sun Grid Engine (SGE)
+---------------------
+
+This example shows a HPC running SGE with 30 CPUs per node.
+
+.. code::
```

Comment:
Maybe we can add a short paragraph about jinja or point the readers to their page so one can look up the syntax. Otherwise people not familiar with this will struggle immensely trying to understand what we are doing here.